src/vivarium/framework/values.py

```diff
@@ -261,7 +271,7 @@ def name(self):
 
     def setup(self, builder):
         self.logger = builder.logging.get_logger(self.name)
-        self.step_size = builder.time.step_size()
+        self.step_size = builder.time.simulant_step_sizes
```

Comment:
I think I missed this in the previous PR (or maybe I just said the wrong thing entirely), but this should be 
```python
self.step_sizes = builder.time.simulant_step_sizes()
```
and in the `TimeInterface` we should have 
```python
def simulant_step_sizes(self) -> Callable:
    return lambda: self._manager.simulant_step_sizes
```
and the same for `simulant_next_event_times`.

Reply:
@rmudambi Why? Just to keep builder's interface consistent w/ callables?

Reply:
done

Reply:
@stevebachmeier yes this is for making our interactions with builder interfaces consistent